diff --git a/plugins/git/git.plugin.zsh b/plugins/git/git.plugin.zsh
index adca1c3..87b31c5 100644
--- a/plugins/git/git.plugin.zsh
+++ b/plugins/git/git.plugin.zsh
@@ -37,7 +37,8 @@ function git_main_branch() {
   
   local remote ref
   
-  for ref in refs/{heads,remotes/{origin,upstream}}/{main,trunk,mainline,default,stable,master}; do
+  local ref
+  for ref in refs/{heads,remotes/{origin,upstream}}/{master,main,trunk,mainline,default,stable}; do
     if command git show-ref -q --verify $ref; then
       echo ${ref:t}
       return 0
diff --git a/themes/robbyrussell.zsh-theme b/themes/robbyrussell.zsh-theme
index cfecfc8..ab21087 100644
--- a/themes/robbyrussell.zsh-theme
+++ b/themes/robbyrussell.zsh-theme
@@ -1,4 +1,4 @@
-PROMPT="%(?:%{$fg_bold[green]%}%1{➜%} :%{$fg_bold[red]%}%1{➜%} ) %{$fg[cyan]%}%c%{$reset_color%}"
+PROMPT="%(?:%{$fg_bold[green]%}%1{➜%} :%{$fg_bold[red]%}%1{➜%} )%D{%H:%M:%S} %{$fg[cyan]%}%c%{$reset_color%}"
 PROMPT+=' $(git_prompt_info)'
 
 ZSH_THEME_GIT_PROMPT_PREFIX="%{$fg_bold[blue]%}git:(%{$fg[red]%}"
